scripts/build

```diff
@@ -18,4 +18,4 @@ if [ "$CROSS" = "true" ] && [ "$ARCH" = "amd64" ]; then
     GOOS=darwin go build -ldflags "$LINKFLAGS" -o bin/rio-darwin ./cli
     GOOS=windows go build -ldflags "$LINKFLAGS" -o bin/rio-windows ./cli
 fi
-CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/rio-controller
+GOOS=linux CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/rio-controller
```

Comment:
This will prob break our local scripts to build controller, but we can work around that for now

Reply:
Yeah, if we have people using script build to run the controller locally but I think most of the scripted flows want to build the controller binary in order to load it into a docker image.